src/Button/button.stories.js

```diff
@@ -19,16 +10,30 @@ const options = [
   'matisse-blue',
   'calico-orange',
 ]
-const defaultValue = 'copper'
 
-export const Default = () => {
-  return (
-    <Button
-      className="min-w-40 md:min-w-40 lg:min-w-48"
-      color={select(label, options, defaultValue)}
-      disabled={boolean('Disabled', false)}
-    >
-      Button
-    </Button>
-  )
+export const Default = args => (
+  <Button className="min-w-40 md:min-w-40 lg:min-w-48" {...args}>
+    Button
+  </Button>
+)
+
+Default.args = {
+  disabled: false,
+  color: defaultColor,
```

Comment:
Any reason you didn't just inline the defaultColor here?

Reply:
changed